src/compression.rs

```diff
@@ -18,6 +18,27 @@ pub(crate) enum CompressionImpl {
     Zstd,
 }
 
+lazy_static::lazy_static! {
+    static ref DEFAULT_GZ_LEVEL: u32 = {
+        std::env::var("AXOASSET_GZ_LEVEL")
+            .ok()
+            .and_then(|val| val.parse().ok())
+            .unwrap_or(6)
```

Comment:
i am trusting you that the default values are good- it may be useful to comment why you picked those, but i wouldn't consider it a blocker

Reply:
6 seems like a good default - checking the `gzip` manpages, both BSD and GNU gzip default to 6. Agree that a comment would be great.

Reply:
So the numeric values you're seeing here is what axoasset _currently_ does — it just wasn't visible in the code until now.

I followed the thread to find out what compression level it was actually applying, and.. the only default that jumps out to me is the .xz one, I think at 9 it's a little extreme.

However, changing the default would be user-facing so we need to talk about it, announce it in advance, provide cargo-dist config to tune it, etc. — all best left for follow-ups.